### PR TITLE
chore: windows 7 deprecation blog post

### PR DESCRIPTION
**Pull request by VerteDinde**
Adds a blog post announcing Windows 7/8 deprecation.

### CODE REVIEW

File: blog/windows-7-deprecation [hunk 1]
```diff
@@ -0,0 +1,46 @@
+---
+title: Farewell, Windows 7 (and 8)
+date: 2022-11-29T00:00:00.000Z
+authors:
+    - name: vertedinde
+      url: 'https://github.com/vertedinde'
+      image_url: 'https://github.com/vertedinde.png?size=96'
+slug: windows-7-deprecation-notice
```
Comment (erickzhao) [inline blog/windows-7-deprecation:8]:
```suggestion
slug: windows-7-to-8-1-deprecation-notice
```

File: blog/windows-7-deprecation [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+Electron will end support of Windows 7, Windows 8 and Windows 8.1 beginning in Electron 23.
+
+---
+
+[In line with Chromium’s deprecation policy](https://support.google.com/chrome/thread/185534985/sunsetting-support-for-windows-7-8-8-1-in-early-2023?hl=en), Electron will end support of Windows 7, Windows 8 and Windows 8.1 beginning in Electron 23. This matches Microsoft's end of support for [Windows 7 ESU](https://docs.microsoft.com/en-US/lifecycle/faq/extended-security-updates) and [Windows 8.1 extended](https://support.microsoft.com/en-us/windows/windows-8-1-support-will-end-on-january-10-2023-3cfd4cde-f611-496a-8057-923fba401e93) on January 10th, 2023.
+
+Electron 22 will be the last Electron major version to support Windows versions older than 10. Windows 7, 8 and 8.1 will not be supported in Electron 23 and later major releases. Older versions of Electron will continue to function on Windows 7, and we will continue to release patches for Electron the 22.x series until May 30 2023, when Electron will end support for 22.x (according to our [support timeline](https://www.electronjs.org/docs/latest/tutorial/electron-timelines)).
+
+## Why Deprecate?
```
Comment (erickzhao) [inline blog/windows-7-deprecation:20]:
```suggestion
## Why deprecate?
```

File: blog/windows-7-deprecation [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+Electron follows the planned Chromium deprecation policy, which will deprecate support in Chromium 109 ([read more about Chromium's timeline here](https://support.google.com/chrome/thread/185534985/sunsetting-support-for-windows-7-8-8-1-in-early-2023?hl=en)). Electron 23 will contain Chromium 110, which won’t support older versions of Windows.
+
+Electron 22, which contains Chromium 108, will thus be the last supported version.
+
+## Deprecation Timeline
```
Comment (erickzhao) [inline blog/windows-7-deprecation:26]:
```suggestion
## Deprecation timeline
```

File: blog/windows-7-deprecation [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+The following is our planned deprecation timeline:
+
+- **December 2022**: The Electron team is entering a quiet period for the holidays
+- **January 2023**: Windows 7 & 8 related issues are accepted for all supported release branches.
+- **February 7, 2023**: Electron 23 is released.
+- **February 8 2023 - May 29 2023**: Electron will continue to accept fixes for supported lines older than Electron 23.
+- **May 30, 2023**: Electron 22 reaches the end of its support cycle.
```
Comment (erickzhao) [inline blog/windows-7-deprecation:34]:
```suggestion
- **December 2022**: The Electron team is entering a quiet period for the holidays
- **January 2023**: Windows 7 & 8 related issues are accepted for all supported release branches.
- **February 7 2023**: Electron 23 is released.
- **February 8 2023 - May 29 2023**: Electron will continue to accept fixes for supported lines older than Electron 23.
- **May 30 2023**: Electron 22 reaches the end of its support cycle.
```

don't mind one way or the other but the date formats should be consistent

File: blog/windows-7-deprecation [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+
+- The Electron team will accept issues and fixes related to Windows 7/8/8.1 for stable supported lines, until each line reaches the end of its support cycle. 
+  - This specifically applies to Electron 22, Electron 21 and Electron 20.
+- New issues related to Windows 7/8/8.1 will be accepted for Electron versions older than Electron 23. 
+  - New issues will not be accepted for any newer release lines.
+- Once Electron 22 has reached the end of its support cycle, all existing issues related to Windows 7/8/8.1 will be closed.
+
+## What's Next
+
+Please feel free to write to us at info@electronjs.org if you have any questions or concerns. You can also find direct support in our official Electron Discord (link).
```
Comment (erickzhao) [inline blog/windows-7-deprecation:46]:
```suggestion
Please feel free to write to us at info@electronjs.org if you have any questions or concerns. You can also find community support in our official [Electron Discord](https://discord.gg/electronjs).
```

**Review by georgexu99 — APPROVED** [review]

**Review by erickzhao — APPROVED** [review]

**Pull request merged**